examples/example2d.rs

```diff
@@ -270,6 +271,21 @@ fn populate_fruit(mut populate: YoleckPopulate<Fruit>, assets: Res<GameAssets>)
     });
 }
 
+fn duplicate_fruit(mut edit: YoleckEdit<Fruit>, mut writer: EventWriter<YoleckDirective>) {
+    edit.edit(|_ctx, data, ui| {
+        if ui.button("Duplicate").clicked() {
+            writer.send(YoleckDirective::spawn_entity(
```

Comment:
Does that work for you, @ZeeQyu?

Reply:
That's brilliant, thanks!

Very clean, well done.